jsonnet/thanos-mixin/alerts/store.libsonnet

```diff
@@ -0,0 +1,88 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'thanos-store.rules',
+        rules: [
+          {
+            alert: 'ThanosStoreGrpcErrorRate',
+            annotations: {
+              message: 'Thanos Store is returning Internal/Unavailable errors.',
+            },
+            expr: |||
+              rate(
+                grpc_server_handled_total{grpc_code=~"Unknown|ResourceExhausted|Internal|Unavailable", %(thanosStoreSelector)s}[5m]
+              ) > 0
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+          },
+          {
+            alert: 'ThanosStoreBucketOperationsFailed',
+            annotations: {
+              message: 'Thanos Store is failing to do bucket operations.',
+            },
+            expr: |||
+              rate(
+                thanos_objstore_bucket_operation_failures_total{%(thanosStoreSelector)s}[5m]
+              ) > 0
```

Comment:
same counter issue as above

Reply:
Maybe we can check AWS SLOs to match this as an example.